### DISCUSSION TITLE
GIDEALED ZC05M - pairing?

### DISCUSSION THREAD

**Discussion by Schmittfinger** [Koenkk/zigbee2mqtt#12228] [Devices]
Hi everybody,

I bought some GIDEALED ZC05M RGB-Controller for LED stripes.
Since February 2022 (I hope) they are supported by zigbee2mqtt (see https://www.zigbee2mqtt.io/devices/ZC05M.html)

How can I pair them to my zigbee2mqtt?
I found some inforrmation about resetting them but I found no information about pairing.

Can someone help me to pair the ZC05M-Controller?

Kind regards and greetings from Germany

Sebastian

**Comment by Schmittfinger** [comment]
I fixed the problem myself. Pressing the reset button sets the device in pairing-mode. I just had some problems with my zigbee-usb-stick.
Release/Stick-in-again my zigbee-usb-device solved my problem and the pairing works.

**Reply by sreeg** [reply]
Im using [Zigbee2MQTT] usb stick and when paired it is showing the device as unsupported.
Below is the device info from "About" tab:
Last seen
N/A
Availability
Online
Device type
Router
Zigbee Model
A11
Zigbee Manufacturer
Light
Support status
Not supported

IEEE Address
0x94deb8fffe169fa1
Network address
0xFD6F
Firmware build date
20210302
Firmware version
3.1.0
Power
Interview completed
True
